docs/source/cookbooks/override-operator-images.md

```diff
@@ -0,0 +1,27 @@
+# Override operator images
```

Comment:
@cjeanner I haven't tested these steps locally but logically the steps look correct.

I would like to add that we have a few possible scenarios for image override:-

1) The user has already built an OpenStack image that they want to override.

2) The user has already built the individual operator image and only wants to rebuild the OpenStack operator including their individual operator changes/image.

3) When the user has made changes for the individual operator (locally) and wants to rebuild both the individual and meta operators' image with their changes.

We are covering scenario 2) with this PR but we need some information about 1) and 3), which can be added in a follow-up (I don't want to block this PR)